wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -95,10 +95,9 @@ class WireGenerator(
     val errorCollector = ErrorCollector()
     val linker = Linker(loader, errorCollector, permitPackageCycles = true, loadExhaustively = true)
 
+    val sourcePaths = setOf(*request.fileToGenerateList.toTypedArray())
```

Comment:
We only want to generate code for this set. This is the subset from `proto_file` that is actually being compiled/generated.

This corresponds to [this field](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/compiler/plugin.proto#L66-L69) in the request.